src/controllers/verifyCommand.ts

```diff
@@ -3,9 +3,11 @@ import { env } from "../typeDefinitions/default.types";
 import { discordTextResponse } from "../utils/discordResponse";
 import { generateUniqueToken } from "../utils/generateUniqueToken";
 import { sendDiscordDm } from "../utils/sendDiscordDm";
+import { sendUserDiscordData } from "../utils/sendUserDiscordData";
 
 export async function verifyCommand(userId: number, env: env) {
   const token = await generateUniqueToken();
+  await sendUserDiscordData(token, userId, env);
```

Comment:
Is this a fire and forget fetch call? If not why are we not handling `error` case here ?

Reply:
please check this